node/Cargo.toml

```diff
@@ -34,6 +33,7 @@ jsonrpc-pubsub = "15.0.0"
 sha3 = { version = "0.8", default-features = false }
 tiny-hderive = { version = "0.3.0", default-features = false }
 tiny-bip39 = {version = "0.6", default-features = false}
+tokio = { version = "0.2.13", features = ["macros", "sync"] }
```

Comment:
Is it ok to have 0.2.13 in node and 0.2.21 in debug/trace?

Reply:
It seems to have changed in master. Dependency resolution should give the same dep version. I'll still change both to 0.2.13 to avoid any issue.